### ISSUE TITLE
Detected a strange `oelint.task.nopythonprefix` warning

### ISSUE THREAD

**Issue by TiagoBeleza** [priv-kweihmann/oelint-adv#915]
Currently using `oelint-adv==9.9.2` and `oelint-parser==8.11.5` and during the run an error appeared for this kind of test case:

Test Case:
`python func() {
   import json
   foo = {
         "foobar": []
   }
}
`

The error `oelint.task.nopythonprefix` is triggered for this function which for me is strange.

Is this expected or not?

**Comment by priv-kweihmann** [comment]
```python
import json foo = { "foobar": [] } 
```

Isn't valid python code.
Maybe your formatting is broken, so I checked

```
python func() { 
    import json
    foo = { "foobar": [] } 
} 
```

Which works absolutely fine with the latest of linter and parser

**Comment by TiagoBeleza** [comment]
Sorry about the formatting.

So I added to `test_class_oelint_task_nopythonprefix.py` this extra test:

```
 'oelint_adv_test.bb':
    '''
    python do_baz() {
        import json
        foo = {
            "bar": []
        }
    }
    ''',
```

And the test fails:
`FAIL: tests/test_class_oelint_task_nopythonprefix.py::TestClassOelintTaskNoPythonPrefix::test_good[input_2-0-oelint.task.nopythonprefix]`

with the warning:
`/tmp/tmpmuciim97/oelint_adv_test.bb:1:warning:oelint.task.nopythonprefix:Tasks that don't contain valid python code (e.g. shell code), should not be prefixed with python in function header [branch:true]`

I am trying to figure it out why this returns a warning
